csharp/resource-overrides/src/ResourceOverrides/ResourceOverrides.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- CDK Construct Library dependencies -->
+    <PackageReference Include="Amazon.CDK" Version="1.18.0" />
+    <PackageReference Include="Amazon.CDK.AWS.EC2" Version="1.18.0" />
+    <PackageReference Include="Amazon.CDK.AWS.S3" Version="1.18.0" />
+    <PackageReference Include="Amazon.CDK.AWS.AutoScaling" Version="1.18.0" />
```

Comment:
```suggestion
    <PackageReference Include="Amazon.CDK" Version="*" />
    <PackageReference Include="Amazon.CDK.AWS.EC2" Version="*" />
    <PackageReference Include="Amazon.CDK.AWS.S3" Version="*" />
    <PackageReference Include="Amazon.CDK.AWS.AutoScaling" Version="*" />
```

Reply:
we want to leave these as wildcard so they reflect the latest version.
This one may be different since it's a reference and the README indicates that it may not build.

what's a good way to ensure these are up to date?